src/interface/src/app/plan/scenario-not-started/scenario-not-started.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-scenario-not-started',
+  templateUrl: './scenario-not-started.component.html',
+  styleUrls: ['./scenario-not-started.component.scss']
+})
+export class ScenarioNotStartedComponent implements OnInit {
```

Comment:
I haven't done it for this one (as I dont know what will be the shape of the data for the charts) but I think this component should be as ` ProjectAreasComponent` , meaning that it should be a dumb component and receive via Input the data already formatted and good to go.